packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts

```diff
@@ -44,17 +46,17 @@ function warnOnAuth(context, map) {
 }
 
 function getTransformerFactory(context, resourceDir) {
-  return async () => {
+  return async options => {
```

Comment:
I think this can be:

```suggestion
  return async (options?: TransformerFactoryArgs) => {
```